.gitignore

```diff
@@ -15,5 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-
-.DS_Store
```

Comment:
@john-kurkowski I would agree, since this wouldn't affect Linux or Windows users.